### PR TITLE
k8s docs - ACLs refactor - Updating terminating gateway documentation to call out updating the role rather than the token with the policy

### PR DESCRIPTION
**Pull request by jmurret**

### CODE REVIEW

File: website/content/docs/k8s/connect/terminating-gateways.mdx
Comment (ishustava) [inline website/content/docs/k8s/connect/terminating-gateways.mdx:150]: should we keep this note? I think `-merge-roles` flag still seems appropriate

**Review by ishustava — APPROVED** [review]
Looks great!!

I left some questions, but won't block on that.

File: website/content/docs/k8s/connect/terminating-gateways.mdx
Comment (jmurret) [reply website/content/docs/k8s/connect/terminating-gateways.mdx:150]: these are only options on updating tokens.

Comment (jmurret) [reply website/content/docs/k8s/connect/terminating-gateways.mdx:150]: https://www.consul.io/commands/acl/tokens/update#command-options has them

https://www.consul.io/commands/acl/role/update#command-options does not

File: website/content/docs/k8s/connect/terminating-gateways.mdx
Comment (jmurret) [reply website/content/docs/k8s/connect/terminating-gateways.mdx:174]: great call!

File: website/content/docs/k8s/connect/terminating-gateways.mdx
Comment (ishustava) [reply website/content/docs/k8s/connect/terminating-gateways.mdx:150]: Oh got it! Thanks for clarifying!

File: website/content/docs/k8s/connect/terminating-gateways.mdx
Comment (david-yu) [inline website/content/docs/k8s/connect/terminating-gateways.mdx:147]:
```suggestion
- Update the existing role to include the new policy
```

**Review by david-yu — APPROVED** [review]

**Comment by hc-github-team-consul-core** [comment]
:cherries: If backport labels were added before merging, cherry-picking will start automatically.

To retroactively trigger a backport after merging, add backport labels and re-run https://circleci.com/gh/hashicorp/consul/628130.

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit 25c32be517b6073ab5f176638821248d88ab454b onto `stable-website` succeeded!

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit 25c32be517b6073ab5f176638821248d88ab454b onto `release/1.12.x` succeeded!

**Pull request merged**
